lib/deprecated/runtime-core/src/vm.rs

```diff
@@ -38,6 +40,9 @@ pub struct Ctx {
     pub data_finalizer: Option<fn(data: *mut c_void)>,
 }
 
+/// We mark `Ctx` as a legacy env that can be passed by `&mut`.
+unsafe impl LegacyEnv for Ctx {}
```

Comment:
Optional: what about `UnsafeMutableEnv`?

Reply:
Yeah, I'm open to a new name, that does sound more descriptive